packages/client/components/TeamPromptMeeting.tsx

```diff
@@ -1,27 +1,43 @@
-import styled from '@emotion/styled'
-import React from 'react'
+import React, {Suspense} from 'react'
 import {Link} from 'react-router-dom'
 
-const Root = styled('div')({
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'center',
-  height: '100%'
-})
-
 import logoMarkPurple from '../styles/theme/images/brand/mark-color.svg'
+import MeetingArea from './MeetingArea'
+import MeetingContent from './MeetingContent'
+import MeetingHeaderAndPhase from './MeetingHeaderAndPhase'
+import MeetingStyles from './MeetingStyles'
+import MeetingTopBar from './MeetingTopBar'
+import PhaseHeaderTitle from './PhaseHeaderTitle'
+import PhaseWrapper from './PhaseWrapper'
 
 interface Props {
   meeting: any
 }
 
 const TeamPromptMeeting = (_props: Props) => {
   return (
-    <Root>
-      <Link title='My Dashboard' to='/meetings'>
-        <img alt='Parabol' src={logoMarkPurple} />
-      </Link>
-    </Root>
+    <MeetingStyles>
+      <MeetingArea>
+        <Suspense fallback={''}>
+          <MeetingContent>
+            <MeetingHeaderAndPhase hideBottomBar={true}>
+              <MeetingTopBar
```

Comment:
I'd avoid using this component here. Standup meeting has totally different styles and I'm not sure we're gaining anything by reusing it here. It's safer to create a `TeamPromptMeetingTopBar` and just make everything match the design. 

Things that are different
1. Back button (non existing in current meeting view)
2. Title (different font size)
3. Avatars group (smaller)
4. Facilitator button (not needed for now, but not present in current meetings)
5. Invite button 
6. More button

that's why I'd just create a new component. as we're here we can try to group components related to this type of meeting in a single directory (so it's easier to see what belongs where). I'd create `components/TeamPrompt/` and put everything related in there.

Reply:
that’s a valid tension! in general, we use this rule to decide whether we want to abstract/reuse something: https://www.notion.so/parabol/How-to-Do-a-Code-Review-daaf03fae3484cf5a1e18174170dadff#598e7a6900f5430ab69f85d88c2f055f 

regarding this particular case, here is what worries me

1. we’ll be refactoring a component used in production by all other meetings, to make it fit the designs of a new, experimental meeting which is under heavy development/experimentation mode. So basically, I’m worried about creating a component with bad interface which in long term will cost us more than creating a new one, for now. here’s a good read about that: https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction 
2. `MeetingTopBar` already has a lot of stuff this meeting currently won’t need, like `toggleDrawer`, `toggleSideBar`, `isCommentUnread`, `isMeetingSidebarCollapsed` and list goes on. 

that said, here’s what I’d do. I'd ask a different question. is there a better way of creating a top bar, in more modular, composable way? if yes, let’s do it and create tasks to refactor existing meetings to use the new component. 

for example, new top bar needs navigation and overflow menu which are not present in other meetings. maybe it’d be good to create a separate component, called `NavigationBar` which would only have the navigation and overflow menu. then, we could use it that way:
```
<NavigationBar>
	<MeetingTopBar>
		…
	</MeetingTopBar>
</NavigationBar>
```
this is just an idea, there might be a better way to do this. 

TLDR, instead of changing the existing component, I’d create a better one, more composable and create tasks to reuse it in other meetings. what do you think @jmtaber129?

Reply:
great! would you like to play with it in this task? once we have a we'll create tasks and handle other meetings separately.

Reply:
Yea, I'll create this new component as part of this task

Reply:
After thinking about this a bit more, I ended up trying out a slightly different approach.  Specifically:
* Export styled components from `MeetingTopBar`
* Use these components to compose a meeting-/phase-specific top bar.  E.g. a `RetroTopBar` for retros that includes the sidebar and `DemoCreateAccountButton`, a `SidebarMeetingTopBar` for any meetings with sidebars that don't need additional customization, etc.

The thinking here is that the core functionality of the current `MeetingTopBar` is pretty basic - render some children (maybe with a sidebar toggle) and render the avatars that were passed in.  However, the scope of the component as an abstraction has increased quite a bit - maybe show a sidebar toggle, show a `DemoCreateAccountButton` for retro demos, show a discussion toggle drawer for sprint poker estimation.  Refactoring other meetings to use this approach would maintain consistent and reusable styles and allow reuse of most existing logic, while also allowing meeting-/phase-level customization.

If this approach were adopted for the other meetings, the refactor would look something like:
* Create a `SidebarHeadingBlock` for any top bars that need sidebar toggles
* Create a `SidebarMeetingTopBar` component that includes the sidebar toggle, and use that in most phases
* Create a `PokerEstimatePhaseTopBar` component that includes the sidebar toggle and discussion drawer toggle, and use that in the poker estimation phase.
* Create a `RetroTopBar` component that includes the sidebar toggle and `DemoCreateAccountButton`, and use that in retro phases.
* Optionally split the existing `MeetingTopBar.tsx` into a few files within a `MeetingTopBar/` directory.

LMK what you think.

(I'll also refactor the `HeadingBlock` to not need the `isMeetingSidebarCollapsed` on the `TeamPromptTopBar` if this sounds good)

Reply:
sounds really good! having multiple building blocks and then being able to construct specific components for each meeting type is the way to go. as you're in the middle of all of these, do you mind creating refactoring tasks with proper descriptions for other meetings? great job!